src/main/java/mate/academy/spring/config/AppConfig.java

```diff
@@ -0,0 +1,38 @@
+package mate.academy.spring.config;
+
+import java.util.Properties;
+import javax.sql.DataSource;
+import org.apache.commons.dbcp2.BasicDataSource;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.orm.hibernate5.LocalSessionFactoryBean;
+
+@Configuration
+@ComponentScan(basePackages = "mate.academy")
+public class AppConfig {
+    @Bean
+    public DataSource getDataSource() {
+        BasicDataSource dataSource = new BasicDataSource();
+        dataSource.setDriverClassName("com.mysql.cj.jdbc.Driver");
+        dataSource.setUrl("jdbc:mysql://localhost:3306/ticket_app?serverTimeZone=UTC");
+        dataSource.setUsername("root");
+        dataSource.setPassword("root");

```

Comment:
Lets move alll db properties in application.properties file

Reply:
done